appcenter-data/Data.js

```diff
@@ -5,10 +5,26 @@ const ReactNative = require('react-native');
 
 const { AppCenterReactNativeData } = ReactNative.NativeModules;
 
+const TimeToLive = {
+    INFINITE: -1,
+    NO_CACHE: 0,
+    DEFAULT: 86400
+};
+
+const DefaultPartitions = {
+    USER_DOCUMENTS = 'user',
+    APP_DOCUMENTS = 'readonly'
+};
+
 const Data = {
-    read(documentId, partition) {
-        return AppCenterReactNativeData.read(documentId, partition).then(result => {
+    TimeToLive,
+    DefaultPartitions,
+    read(documentId, partition, ttl) {
+        if(ttl === undefined) {
```

Comment:
this if does not look its formatted correctly
```suggestion
        if (ttl === undefined) {
```
I am curious why eslint didnt see it.

Reply:
For some reason working at the root of the project from VSCode doesn't pick it up and eslint errors. If I set my workspace to a specific project eslint errors will show up. I'll be sure to run the eslint script from now on.